ts/store/actions/persistedPreferences.ts

```diff
@@ -17,9 +17,9 @@ export const serviceAlertDisplayedOnceSuccess = createStandardAction(
   "SERVICE_ALERT_DISPLAYED_ONCE_SUCCESS"
 )<{ wasServiceAlertDisplayedOnce: boolean }>();
 
-export const pagoPaQAEnvironmentEnabledSuccess = createStandardAction(
-  "PAGOPA_ENV_AS_QA"
-)<{ isPagoPAQAEnabled: boolean }>();
+export const pagoPaTestEnvironmentEnabledSuccess = createStandardAction(
+  "PAGOPA_ENV_AS_Test"
```

Comment:
all caps and should be named after the action: `PREFERENCES_PAGOPA_TEST_ENVIRONMENT_SET_ENABLED `